src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserTest.java

```diff
@@ -151,4 +152,20 @@ public void should_reinvite_super_user_within_one_hour() {
             assertThat((String) reinviteUserResponse.get("errorDescription")).contains(String.format("The request was last made less than %s minutes. Please try after some time"), resendInterval);
         }
     }
+
+    // should not re invite when user does not exists in organisation
+    @Test
+    public void should_return_403_when_reinvited_user_not_exists_in_same_organisation() {
+        if (resendInviteEnabled) {
+
+            OrganisationCreationRequest organisationCreationRequest1 = someMinimalOrganisationRequest().build();
```

Comment:
could you please make it more specific like internal and external scenario

Reply:
done